### DISCUSSION TITLE
OpenSSL bundled with MinGit

### DISCUSSION THREAD

**Discussion by alenmeister** [git-for-windows/git#3745] [Q&A]
Howdy.

Is there any way to include OpenSSL in the MinGit releases? 
I couldn't find it in the `bin` folder, sadly.

**Comment by dscho** [comment]
OpenSSL, the library, is included.

Since the "Min" in MinGit stands for "minimal", in the sense that everything that is not needed by Git is subject to be excluded from MinGit, including the `openssl.exe` file would be contrary to the goal.

**Reply by alenmeister** [reply]
That makes total sense. Would it still be possible to include it, though? By manually copying over the necessary binaries? Or has there been changes to the MinGit code base that would prevent me from using it?

**Reply by dscho** [reply]
You can always copy directly from the corresponding PortableGit.

**Reply by alenmeister** [reply]
Awesome. Other than `openssl.exe`, what else might be needed?

**Comment by nicoleta-scrimint** [comment]
Hi!

In the Git for Windows version 2.38.1.windows.1, I observed that OpenSSL version 1.1.1.17 is included in the path "GitInstallationPath\mingw64\bin\openssl.exe".
Will Git for Windows deliver a new version with the new OpenSSL versions, if it contains security fixes?

Thank you!

**Reply by dscho** [reply]
> Will Git for Windows deliver a new version with the new OpenSSL versions, if it contains security fixes?

Yes. For example, we [just updated to OpenSSL v1.1.1s](https://github.com/git-for-windows/build-extra/commit/18eab3abde7fa289dbf6678dd6e2b5526c84c74f) and the next official build (as well as the [snapshots](https://wingit.blob.core.windows.net/files/index.html)) will come with those fixes.
